site/includes/sign-on-off.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+	<div class="row">
+{{#is signed "off"}}
```

Comment:
That is not the interface...That is for application that provides links to the sign on interface. You even did it this way in #6232

P.S. your link doesn't work